content/blog/sivayanama-simplifying-open-api-specification-with-reusable-common-data-models.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Simplifying OpenAPI specification with reusable common data models
+date: 2023-08-26T08:06:19.141Z
+author: BalaSubramanian Vetrivel
+authorimage: /img/vetrivel-balasubramanian1-photo.jpg
+disable: false
+tags:
+  - API, OpenAPI, OpenAPI specification, OpsRamp API, Yaml
+---
+The [OpenAPI specification](https://www.openapis.org)  is one of the most widely followed API contracts. It is language-agonistic.
+With the help of these OpenAPI specifications, clients can understand the API and invoke them without having access to the code or worrying about the implementation details. 
+
+At times, this open specification file is becoming too complex to manage and understand. In this article, I will discuss the techniques to simplify the open specification with loosely coupled reusable data models.
```

Comment:
Suggest rephrasing to:
"At times, this open specification file can be too complex to manage and understand."

By the way, I do believe you are referring to this, correct? https://swagger.io/specification/   https://en.wikipedia.org/wiki/OpenAPI_Specification